public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -1,8 +1,13 @@
-include ../_util-fns
+block includes
+  include ../_util-fns
+  - var _JavaScript = 'JavaScript';
+  //- Double underscore means don't escape var, use !{__var}.
+  - var __chaining_op = '<code>;</code> or <code>,</code>';
+  - var __new_op = '<code>new</code>';
+  - var __objectAsMap = 'object';
 
-// #docregion intro
 :marked
-  Our Angular application manages what the user sees and does through the interaction of a Component class instance (the *component*) and its user-facing template.
+  Our Angular application manages what the user sees and it achieves this through the interaction of a Component class instance (the *component*) and its user-facing template.
```

Comment:
It's good to change the original sentence, since it was imprecise, but the new version is rather run-on. How about:

Our Angular application manages what the user sees and can do, achieving this through the...